site/en/docs/extensions/mv3/match_patterns/index.md

```diff
@@ -3,159 +3,86 @@ layout: 'layouts/doc-post.njk'
 title: 'Match patterns'
 seoTitle: 'Chrome Extensions match patterns'
 date: 2012-09-18
-updated: 2023-07-28
+updated: 2023-08-31
 description: Understanding URL match patterns in Chrome extensions.
 ---
 
-## Overview {: #overview }
-
-Extensions use match patterns to specify a group of URLs. They are mainly used to declare [host permissions][host-permissions] along with other permissions to use certain Chrome APIs or when injecting [content scripts][content-scripts]. Extensions also use match patterns in:
-
-- [`"web_accessible_resources.matches"`][war]
-- [`"externally_connectable.matches"`][ext-connect]
-
-## What is a match pattern? {: #what }
-
-A match pattern is a URL that begins with a permitted scheme, separated by a `://`, followed by a host and path. It can contain wildcard (`*`) characters. Most match patterns are composed of three parts:
+A match pattern is a URL with the following structure, used to specify a group of URLs:
 
 ```text
 &lt;scheme&gt;://&lt;host&gt;/&lt;path&gt;
 ```
 
-Each part can use wildcards `*`. Below is a detailed description:
+**scheme**: Must be one of the following, separated from the rest of the pattern using a double slash (`//`):
+
+- `http`
+- `https`
+- A wildcard `*`, which matches only `http` or `https`
+- `file`
+- `ftp`
+- `chromeui`
+- `extension`
+- `filesystem`
+- `ws`
+- `wss`
+- `data`
+- `uuid_in_package`
```

Comment:
```suggestion
- `http`
- `https`
- A wildcard `*`, which matches only `http` or `https`
- `file`
```
@wizardofdocs @jpmedley 
Could we apply this change before merging? [Here's the conversation where we agreed to simplify this list](https://github.com/GoogleChrome/developer.chrome.com/pull/7215#discussion_r1352008813)